commands/cluster-getslotmigrations.md

```diff
@@ -0,0 +1,94 @@
+`CLUSTER GETSLOTMIGRATIONS` returns an array of information about in-progress
+and recently completed
+[atomic slot migrations](../topics/atomic-slot-migration.md).
+
+Each job previously started by
+[`CLUSTER MIGRATESLOTS`](cluster-migrateslots.md) creates a single entry. The
+number of visible slot migration entries depends on the configured
+`cluster-slot-migration-log-max-len`. After the limit is reached, the oldest
+inactive entry is removed. Note that active slot migrations will always be
+visible through `CLUSTER GETSLOTMIGRATIONS`, even if there are more entries than
+the configured limit.
+
+Information about the slot migration operations is stored in-memory and is not
+persisted across restarts. Slot migration entries are visible on the primary
+node of both the target and source shard and the replica nodes of the target
+shard.
+
+The following information is reported for each slot migration entry:
+
+- `name` -- A unique 40 byte name for the slot migration
+- `operation` -- The operation performed by the slot migration job on this node
+  (either `EXPORT` or `IMPORT`)
+- `slot_ranges` -- The range(s) of slots being migrated, with both start and end
+  inclusive. The start and end slot are separated by `-` in each range, and
+  multiple ranges are joined together with ` `
```

Comment:
This space is only visible if code text is rendered with a differnt background color.

I think we need to display it more clearly, with quotes and/or spell out the word space or space-separated.

```suggestion
  multiple ranges are joined together with a space ("` `").
```

Alternatively rewrite the whole sentence as

*A space-separated list of slot ranges being migrated ...*